2024-02-23 12:08:18,920 INFO    MainThread:137005 [wandb_setup.py:_flush():76] Current SDK version is 0.16.3
2024-02-23 12:08:18,920 INFO    MainThread:137005 [wandb_setup.py:_flush():76] Configure stats pid to 137005
2024-02-23 12:08:18,920 INFO    MainThread:137005 [wandb_setup.py:_flush():76] Loading settings from /home/cal314-1/.config/wandb/settings
2024-02-23 12:08:18,920 INFO    MainThread:137005 [wandb_setup.py:_flush():76] Loading settings from /media/cal314-1/9E044F59044F3415/Marcin/noisy_labels/wandb/settings
2024-02-23 12:08:18,920 INFO    MainThread:137005 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-02-23 12:08:18,921 INFO    MainThread:137005 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-02-23 12:08:18,921 INFO    MainThread:137005 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'Kod/train.py', 'program_abspath': '/media/cal314-1/9E044F59044F3415/Marcin/noisy_labels/Kod/train.py', 'program': '/media/cal314-1/9E044F59044F3415/Marcin/noisy_labels/Kod/train.py'}
2024-02-23 12:08:18,921 INFO    MainThread:137005 [wandb_init.py:_log_setup():526] Logging user logs to /media/cal314-1/9E044F59044F3415/Marcin/noisy_labels/wandb/run-20240223_120818-tvvx1xq8/logs/debug.log
2024-02-23 12:08:18,921 INFO    MainThread:137005 [wandb_init.py:_log_setup():527] Logging internal logs to /media/cal314-1/9E044F59044F3415/Marcin/noisy_labels/wandb/run-20240223_120818-tvvx1xq8/logs/debug-internal.log
2024-02-23 12:08:18,921 INFO    MainThread:137005 [wandb_init.py:init():566] calling init triggers
2024-02-23 12:08:18,921 INFO    MainThread:137005 [wandb_init.py:init():573] wandb.init called with sweep_config: {}
config: {'epochs': 300, 'batch_size': 22, 'lr': 0.0001, 'annotator': 2, 'model': 'smpUNet++', 'augmentation': False, 'loss': 'CrossEntropyLossWeight', 'optimizer': 'Adam', 'scheduler': 'CosineAnnealingLR', 'place': 'lab', 'mode': 'intersection_and_union'}
2024-02-23 12:08:18,921 INFO    MainThread:137005 [wandb_init.py:init():616] starting backend
2024-02-23 12:08:18,921 INFO    MainThread:137005 [wandb_init.py:init():620] setting up manager
2024-02-23 12:08:18,925 INFO    MainThread:137005 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2024-02-23 12:08:18,926 INFO    MainThread:137005 [wandb_init.py:init():628] backend started and connected
2024-02-23 12:08:18,931 INFO    MainThread:137005 [wandb_init.py:init():720] updated telemetry
2024-02-23 12:08:18,940 INFO    MainThread:137005 [wandb_init.py:init():753] communicating run to backend with 90.0 second timeout
2024-02-23 12:08:19,545 INFO    MainThread:137005 [wandb_run.py:_on_init():2262] communicating current version
2024-02-23 12:08:19,750 WARNING MainThread:137005 [wandb_init.py:init():1198] interrupted
Traceback (most recent call last):
  File "/home/cal314-1/miniconda3/envs/marcinv2/lib/python3.11/site-packages/wandb/sdk/wandb_init.py", line 1176, in init
    run = wi.init()
          ^^^^^^^^^
  File "/home/cal314-1/miniconda3/envs/marcinv2/lib/python3.11/site-packages/wandb/sdk/wandb_init.py", line 802, in init
    run._on_init()
  File "/home/cal314-1/miniconda3/envs/marcinv2/lib/python3.11/site-packages/wandb/sdk/wandb_run.py", line 2266, in _on_init
    version_result = version_handle.wait(timeout=30)
                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/cal314-1/miniconda3/envs/marcinv2/lib/python3.11/site-packages/wandb/sdk/lib/mailbox.py", line 283, in wait
    found, abandoned = self._slot._get_and_clear(timeout=wait_timeout)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/cal314-1/miniconda3/envs/marcinv2/lib/python3.11/site-packages/wandb/sdk/lib/mailbox.py", line 130, in _get_and_clear
    if self._wait(timeout=timeout):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/cal314-1/miniconda3/envs/marcinv2/lib/python3.11/site-packages/wandb/sdk/lib/mailbox.py", line 126, in _wait
    return self._event.wait(timeout=timeout)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/cal314-1/miniconda3/envs/marcinv2/lib/python3.11/threading.py", line 629, in wait
    signaled = self._cond.wait(timeout)
               ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/cal314-1/miniconda3/envs/marcinv2/lib/python3.11/threading.py", line 331, in wait
    gotit = waiter.acquire(True, timeout)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
KeyboardInterrupt
2024-02-23 12:08:22,333 WARNING MsgRouterThr:137005 [router.py:message_loop():77] message_loop has been closed
